website/pages/docs/commands/validate.mdx

```diff
@@ -33,13 +33,11 @@ Errors validating build 'vmware'. 1 error(s) occurred:
 - `-syntax-only` - Only the syntax of the template is checked. The
   configuration is not validated.
 
-- `-except=foo,bar,baz` - Builds all the builds and post-processors except
-  those with the given comma-separated names. Build and post-processor names
-  by default are the names of their builders, unless a specific `name`
-  attribute is specified within the configuration. A post-processor with an
-  empty name will be ignored.
+- `-except=foo,bar,baz` - Validates all the builds except those with the
+  comma-separated names. Build names by default are the names of their
+  builders, unless a specific `name` attribute is specified within the configuration.
 
-- `-only=foo,bar,baz` - Only build the builds with the given comma-separated
+- `-only=foo,bar,baz` - Only validate the builds with the given comma-separated
   names. Build names by default are the names of their builders, unless a
   specific `name` attribute is specified within the configuration.
```

Comment:
should we simply remove these for now ?

Reply:
Seeing as the only and except flags work as expected for validating certain build configurations of a template file, and have previously worked for JSON template files. I think it is okay to leave the options as they are. Especially, since `packer build template.json` will skip over any invalid builders and continue with the valid ones.

 But I can see the confusion if it doesn't work for all parts of the the validate command namely the fixer check. So I can go either way.

Reply:
oh right ! I didn't get the full picture here 🤔